README.md

```diff
@@ -1,36 +1,70 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15555388&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
```

Comment:
Software engineering is a discipline focused on the systematic development of high-quality software products using engineering principles, methods, and tools. Software engineering plays a crucial role in the technology industry by enabling the creation of reliable, scalable, and innovative software solutions that meet user needs.
Some of the importance of software engineering are 
Building reliable software: It is the company’s responsibility to deliver software products on schedule and to address any defects that may exist.
Effectiveness: Effectiveness results from things being created in accordance with the standards.
Reduces complexity: Large challenges are broken down into smaller ones and solved one at a time in software engineering. Individual solutions are found for each of these issues.
Productivity: Because it contains testing systems at every level, proper care is done to maintain software productivity.